src/NumericalAlgorithms/Interpolation/Actions/InterpolateToTarget.hpp

```diff
@@ -68,13 +68,15 @@ struct InterpolateToTarget {
     // in several steps:
     const auto& element_coord_holder = element_coord_holders.at(element_ids[0]);
 
+    using compute_items_in_new_box = tmpl::list_difference<
```

Comment:
is this tested with a an item already in the box?

Reply:
I'm adding such a test now...

Reply:
Could you point us to the place where it's tested? That'll make it easier to review :)